ansible/roles/contiv/tasks/netplugin.yml

```diff
@@ -0,0 +1,81 @@
+- name: ensure pluginDir exists
+  file: path=/usr/libexec/kubernetes/kubelet-plugins/net/exec recurse=yes state=directory
```

Comment:
Right, but we will need to use an Ansible role outside of contrib that creates the dir using cloud-init to support coreOS. In general, we want to eliminate as much hard-coding in tasks as possible.